docs/faq/index.md

```diff
@@ -0,0 +1,93 @@
+# Frequently asked questions
+
+## General questions
+
+### What does the GitOps Toolkit mean for Flux?
+
+Flux v1 is a monolithic do-it-all operator, the GitOps Toolkit separates the functionalities into specialized controllers.
```

Comment:
Style alert! Either use a conjunction (i.e., "and") or a semicolon to join these clauses together. Or better I think, just have two sentences.

Reply:
```suggestion
Flux v1 is a monolithic do-it-all operator; the GitOps Toolkit separates the functionalities into specialized controllers.
```